mne/tests/test_ica.py

```diff
@@ -0,0 +1,24 @@
+import mne
```

Comment:
Can you move this to `mne/preprocessing/tests/test_ica.py`? And maybe also if possible check how we keep computation time low there and do the same here (e.g., cropping the raw instance). We want tests to run as quickly as possible!